src/offices/_single.html

```diff
@@ -100,22 +101,30 @@ <h2 class="u-visually-hidden">
             {% if resource.icon %}
             <div class="media_image-container
                         media_image-container__wide-margin">
-                <img class="media_image u-centered-on-mobile"
-                     width="150"
+                <img class="media_image
```

Comment:
So sounds like we should remove resource icon and subscription on the front-end and the backend? @kurtw Can you open an issue where that's applicable on the backend? Front-end issue opened at https://github.com/cfpb/cfgov-refresh/issues/873